File: app_kotlin/src/main/java/vlab/android/architecture/repository/source/remote/GitHubApi.java
package vlab.android.architecture.repository.source.remote;

import io.reactivex.Observable;
import retrofit2.http.GET;
import retrofit2.http.Header;
import vlab.android.architecture.repository.source.remote.response.UserResponse;

public interface GitHubApi {
    @GET("user")
    Observable<UserResponse> login(@Header("Authorization") String authorization);
}

File: app_kotlin/src/main/java/vlab/android/architecture/repository/GithubApi.kt
package vlab.android.architecture.repository

import retrofit2.Call
import retrofit2.http.GET
import retrofit2.http.Header

/**
 * Created by Vinh.Tran on 8/6/18.
 **/
interface GithubApi {

    @GET("user")
    fun login(@Header("Authorization") auth : String): Call<UserResponse>

}